modules/ROOT/pages/_partials/cloudhub2-apps.adoc

```diff
@@ -63,11 +63,12 @@ Deploys the application specified in `<appID>` to the deployment target specifie
 [%header,cols="12a,53a,35a"]
 |===
 |Value |Description | Example
-|`artifactId` |Artifact ID of the application retrieved from Exchange. | `mule-test-plugin`
-|`deploymentTargetId` |ID of the deployment target. +
-You can get this ID directly from Runtime Manager.  | `cloudhub-us-east-1`
-|`name` |Name of the application to deploy. | `testcloudhub2app`
-|`runtimeVersion` |Runtime version of the deployment target.|`4.4.0`
+|`--artifactId` |Artifact ID of the application retrieved from Exchange. | `--artifactId mule-test-plugin`
+|`--deploymentTargetId` |ID of the deployment target. +
+You can get this ID directly from Runtime Manager.  | `--deploymentTargetId cloudhub-us-east-1`
+|`--name` |Name of the application to deploy. | `--name testcloudhub2app`
+|`--runtimeVersion` |Runtime version of the deployment target.|`--runtimeVersion 4.4.0`
+|`--secureProperty` | Sets an encrypted property. | `--secureProperty camtest2:3 d1519c7b-16ac-45f3-a771-5bcdcf23dda4`
```

Comment:
can we make it more general ?

Reply:
like:

`--secureProperty

Sets a secure property.
Format: name:value

Example: --secureProperty secureTestProperty:true`